modules/OCPPConfiguration/manifest.yaml

```diff
@@ -0,0 +1,33 @@
+description: This module reacts to OCPP configuration changes.
+config:
+  monitor_variables:
+    description: Comma separated list of variables to monitor
+    type: string
+    default: >
+      AllowOfflineTxForUnknownId,AuthorizationCacheEnabled,AuthorizeRemoteTxRequests,
+      BlinkRepeat,ClockAlignedDataInterval,ConnectionTimeOut,ConnectorPhaseRotation,
+      HeartbeatInterval,LightIntensity,LocalAuthorizeOffline,LocalPreAuthorize,
+      MaxEnergyOnInvalidId,MeterValuesAlignedData,MeterValuesSampledData,
+      MeterValueSampleInterval,MinimumStatusDuration,ResetRetries,
+      StopTransactionOnEVSideDisconnect,StopTransactionOnInvalidId,StopTxnAlignedData,
+      StopTxnSampledData,TransactionMessageAttempts,TransactionMessageRetryInterval,
+      UnlockConnectorOnEVSideDisconnect,WebsocketPingInterval
+  user_config_path:
```

Comment:
@hikinggrass : Can the user-config path be accessed from within a module?

Reply:
No, you also don't get the path to the original config. Since this is just a temporary solution until the config service in everest-framework is ready it's fine to do it like this